app/controllers/remove-user-product-ctrl.js

```diff
@@ -0,0 +1,64 @@
+'use strict';
+/* eslint-disable no-console */
+
+const prompt = require('prompt');
+const { red, blue } = require('chalk');
+
+let { dbCheckForProductSales, dbDeleteProduct, dbGetAllProductsByUser } = require('../models/Product.js');
+let { dbDeleteOpenOrderByProduct } = require('../models/Order-Product.js');
+
+module.exports.removeUserProduct = user_id => {
+	return new Promise((resolve, reject) => {
+		dbGetAllProductsByUser(user_id).then(data => {
+			if (data.length == 0) {
+				return reject(`${red('Customer has no products')}`);
+			}
+			let productArr = [];
+			console.log(`Customer #${user_id} - all Products:`);
+			data.forEach((product, index) => {
+				productArr.push(product);
+				console.log(`${index + 1}. ${product.title} (#${product.id})`);
+			});
+			prompt.get(
+				[
+					{
+						name: 'number',
+						description: 'Please select Product'
+					}
+				],
+				function(err, choice) {
+					if (err) {
+						return reject(err);
+					} else {
+						let productId = productArr[choice.number - 1].id;
```

Comment:
It throws an error and kicks you out if you select choice that is not on the list. Instead print an error msg?